demos/coreHTTP/http_demo_s3_download_multithreaded.c

```diff
@@ -147,6 +144,10 @@
     #define httpexampleMAX_DEMO_COUNT    ( 3 )
 #endif
 
+#ifndef httpexampleTASK_STACK_SIZE
+    #define httpexampleTASK_STACK_SIZE    ( configMINIMAL_STACK_SIZE * 2 )
```

Comment:
Why is this the right number?

Reply:
Using the default value of `configMINIMAL_STACK_SIZE` in some boards is causing stack overflow failure in some boards (for example Nuvoton that uses 256 bytes for the macro). This PR is not changing the value of the HTTP task stack size (the diff is just a re-positioning of the config). I wanted to remove the extra allocation of the stack size for the HTTP task as the task uses minimal stack variables, and thus, I'm not completely sure about whether the Queue and direct-to-task notification calls are causing the `configMINIMAL_STACK_SIZE` for some boards to be insufficient